### DISCUSSION TITLE
Error 1006 When executing Input Image, No Swap Issue

### DISCUSSION THREAD

**Discussion by LearningwithAI1** [lllyasviel/Fooocus#3353] [Q&A]
Hi,

I am new to this but i thoroughly went through the trouble shooting steps and have no swap issue (attached) 
![Virtual Memory](https://github.com/user-attachments/assets/43642465-3232-49b3-9176-b8c97dc94881)

When executing Input Image with Face Swap, i get 1006 in my command prompt and "Error
Connection errored out." 

I am however able to generate normal images fast with my NVIDIA RTX A3000 Laptop GPU.

Receiving same error when running through google colab

Please let me know the next steps I can take.  Appreciate your help with this.

**Comment by mashb1t** [comment]
For colab this is expected depending on the size of the input image, see readme.

> some "big" features like image prompt may cause free-tier Colab to disconnect

Please double-check if Fooocus and the swap drive are the same, as this solved the issue for some other people. Im some cases this issue has also been solved by manually setting the swap to a fixed size on the same drive where Fooocus is located.

**Comment by LearningwithAI1** [comment]
1. Foocus and the Swap drive are the same.
2. I tried manually setting initial and maximum size to 40000 and 60000 as i saw that worked for someone here on a seperate discussion thread; however, i have no luck with that.

Is there anything else i can try?
